packages/react-native-session-replay/ios/Sources/RCTTextViewRecorder.swift

```diff
@@ -95,11 +95,13 @@ internal struct RCTTextViewWireframesBuilder: SessionReplayNodeWireframesBuilder
 
     let DEFAULT_FONT_COLOR = UIColor.black.cgColor
 
+    // Clipping should be 0 to avoid the text from overflowing when the
+    // numberOfLines prop is used.
     private var clip: SRContentClip {
```

Comment:
I think we can drop the clip in that case

Reply:
We could... but keeping it ([when it will be applied in the player](https://github.com/DataDog/web-ui/pull/114413)) will enable us to handle ellipsis much better